sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -7,7 +7,7 @@ from form documents. It includes the following main functionalities:
 * Prebuilt receipt model - Recognize data from sales receipts using a prebuilt model.
 * Prebuilt invoice model - Recognize data from USA sales invoices using a prebuilt model.
 * Prebuilt business card model - Recognize data from business cards using a prebuilt model.
-* Prebuilt ID document model - Recognize data from ID documents using a prebuilt model.
+* Prebuilt identity document model - Recognize data from identity documents using a prebuilt model.
```

Comment:
just curious - was this the only spot on the README that needed update? I think there were multiple references for both .NET and Python that needed to be changed.

Reply:
It seems we haven't updated the README for a long time.  It is updated now